src/material/stepper/_stepper-theme.scss

```diff
@@ -15,6 +16,25 @@
   $warn: map.get($config, warn);
 
   .mat-step-header {
+    @include a11y.high-contrast(active, off) {
+      margin-bottom: 1px;
+      margin-top: 1px;
```

Comment:
Why do we need this margin? In general I've been trying to avoid changing any layout properties for high contrast mode.

Reply:
Yeah I was pretty hesistant about this, but removing it causes slight weird overlapping (like the overhead text when typing into the input) over some of the outlines. I am not sure how big of an issue that is but I ended up removing it.